production/helm/values.yaml

```diff
@@ -6,6 +6,8 @@ serviceAccount:
   create: true
   name:
 
+dataRoot: /var/lib/docker
```

Comment:
Add this key to Promtail part of the config as it's acting only on promtail config